deploy/rbac.yaml

```diff
@@ -32,7 +32,11 @@ rules:
   - statefulsets
   verbs:
   - "*"
-
+- apiGroups:
```

Comment:
do we need that? The operator doesn't do anything with rolebindings

Reply:
will remove